src/news_gui.cpp

```diff
@@ -368,8 +370,12 @@ struct NewsWindow : Window {
 
 		this->CreateNestedTree();
 
+		bool has_vehicle_id = std::holds_alternative<VehicleID>(ni->ref1);
+		NWidgetStacked *nwid_sel = this->GetWidget<NWidgetStacked>(WID_N_SHOW_GROUP_SEL);
+		if (nwid_sel != nullptr) nwid_sel->SetDisplayedPlane(!has_vehicle_id ? 0 : SZSP_NONE);
```

Comment:
Why are you negating the check for  `has_vehicle_id` here? Am I misreading this or is this backwards?